src/runtime/builder.rs

```diff
@@ -241,8 +241,14 @@ impl Builder {
                 });
             })
             .custom_park(move |worker_id| {
+                // Create a new reactor
+                let reactor = Reactor::new().unwrap(); // TODO(stjepang): remove unwrap
```

Comment:
This is a tricky `unwrap()`. I think we have two options here:

1. Change the signature of `custom_park` so that the closure returns an `io::Result<P>`.
2. Construct reactors and timers outside the threadpool builder. For that, we should probably make the inner field of `WorkerId` public and guarantee that IDs are always numbers in `0..core_threads`.

Which one do you prefer?